shells/pipes-shell/source/pipe.js

```diff
@@ -50,17 +48,15 @@ export const initPipe = async (client, paths, storage) => {
 
 // TODO(sjmiles): must be called only after `window.ShellApi` is initialized
 export const initArcs = async (storage, bus) => {
-  // marshal ingestion arc
```

Comment:
@sjmiles Scott, please, hold off reviewing. 
sorry about this, I remember now that I shouldn't be just removing this - I'll send a commit shortly to restore the sharing functionality.

Reply:
updated the PR, PTAL